packages/angular/projects/angular-sdk/CHANGELOG.md

```
@@ -1,5 +1,12 @@
 # Changelog
 
+### Dependencies
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)